base_layer/core/src/transaction_protocol/transaction_initializer.rs

```diff
@@ -181,7 +181,8 @@ impl SenderTransactionInitializer {
                             .change_secret
                             .as_ref()
                             .ok_or("Change spending key was not provided")?;
-                        self.with_output(UnblindedOutput::new(v, change_key.clone(), None));
+                        let change_key = change_key.clone();
```

Comment:
This is a weird one to me but I guess clippy gets what it wants.

Reply:
That one actually was potentially a more serious issue -- not just cosmetic; related to making a immutable borrow after a mutable one (or something like that)